pages/common/react-native.md

```diff
@@ -0,0 +1,36 @@
+# react-native
+
+> A framework for building native apps with React.
+> More information: <https://reactnative.dev>.
+
+- Initialize a new React Native project in a directory of the same name:
+
+`react-native init {{project_name}}`
+
+- Start the metro bundler:
+
+`react-native start`
+
+- Start the metro bundler with a clean cache:
+
+`react-native start --reset-cache`
+
+- Build your application and start it on a connected Android device or emulator:
```

Comment:
Generally we try to avoid the 2nd person (e.g. `you`, `your`, etc) in tldr-pages :slightly_smiling_face:
```suggestion
- Build the current application and start it on a connected Android device or emulator:
```